examples/ASDKgram/Sample/PhotoFeedNodeController.m

```diff
@@ -192,7 +192,7 @@ - (void)tableView:(ASTableView *)tableView willBeginBatchFetchWithContext:(ASBat
 - (void)resetAllData
 {
   [_photoFeed clearFeed];
```

Comment:
Also change the other ASTableDelegate methods to `tableNode:numberOfRowsInSection:` etc.